doc/rst/developer/adding-a-comm-layer.rst

```diff
@@ -0,0 +1,62 @@
+=============================
+Adding Communication Layer
+=============================
+This document briefly describes how to add a new communication layer
+to the runtime.
+
+To implement a new comm layer called ``speedy``, add ``comm/speedy`` to the
+build as described in ``adding-a-runtime-component.rst``.
+
+The comm layer is responsible for setting the linker (LD) for the
+generated code, and should do so in ``runtime/etc/Makefile.comm-speedy``.
+Typically LD should be set to ``CXX`` or a ``C++-compatible`` linker in order
+to ensure that standard C++ libraries are linked in to support
+third-party packages that may use C++ (like ``re2``).
+
+The following header files define the comm layer API and also serve as
+documentation for the core API:
+
+- *runtime/include/chpl-comm.h*
+- *runtime/include/chpl-comm-locales.h*
```

Comment:
```suggestion
- ``runtime/include/chpl-comm-locales.h``
```

Reply:
Hey @ben-albrecht, Sorry for the delay. I have updated the code as per your suggestions.